spec/features/display_peeps_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'display peeps' do
+  scenario 'navigate to page and list peeps from database' do
+    visit '/peeps'
+    expect(page).to have_content("This is a peep!")
+  end
```

Comment:
I found it a bit confusing that there's nothing explicitly adding a peep to the DB here. I later found that you have a nice helper method which is called in `spec_helper`.

It's neat, but probably easier for people to follow along if you call that method here, in your test. You could also consider refactoring that helper method to take one arg, which would be the message content, so that it's really easy to see what's going on right from here.